src/main/java/com/bloomtech/asylumrgbea/controllers/AsylumCaseController.java

```diff
@@ -33,11 +41,31 @@ public PageResponseDto getPageOfCases(@RequestBody AsylumCaseRequestDto asylumCa
     /**
      * As the point of entry this method is mapped to the show-all end point and provides all ResponseDtos.
      * @return An Iterable of AsylumCaseResponseDto or an ErrorDto with a message.
+     *
+     * This end-point has been deprecated at this point in development. Removed @GetMapping annotation
+     * to obscure this endpoint from public access.
      */
-    @GetMapping
+    @Deprecated
```

Comment:
Let's remove deprecated endpoints from the codebase.

Reply:
Removed the endpoints.